addons/udes_stock_picking_batch/models/stock_picking_batch.py

```diff
@@ -391,15 +424,21 @@ def _populate_next_tasks(
             priority_ml = move_lines._determine_priority_skipped_moveline(
                 skipped_product_ids, skipped_move_line_ids
             )
-            task = self._populate_next_task(move_lines, task_grouping_criteria, priority_ml)
+            task = self._populate_next_task(
+                move_lines, task_grouping_criteria, priority_ml
+            )
             task["tasks_picked"] = have_tasks_been_picked
             tasks.append(task)
             if limit and len(tasks) >= limit:
                 break
-            move_lines = move_lines.filtered(lambda ml: ml.id not in task["move_line_ids"])
+            move_lines = move_lines.filtered(
+                lambda ml: ml.id not in task["move_line_ids"]
+            )
         return tasks
 
-    def _populate_next_task(self, move_lines, task_grouping_criteria, priority_ml=False):
```

Comment:
Nit: Lots of formatting but this line is 90 characters in length, the rule as far as I am aware is 100 and then it is formatted. Can you check your black formatting script? It should use 100 as the line maximum length.

Reply:
The black formatting script was checked and it is now set at 100 a line maximum length

Reply:
Can you run it again? I tried running it on the file in this commit and the output is different than what's in the last commit so it needs to be reformatted. If it doesn't change then the setting for line length isn't working.